index.d.ts

```diff
@@ -1,29 +1,29 @@
 import * as React from 'react';
 
 type Function = () => void;
+type UpdateFunction = (newEnd?: number) => void;
 
 interface CallbackProps {
   onEnd?: (providedFn: {
     pauseResume: Function;
```

Comment:
```suggestion
    pauseResume: () => void;
```

Reply:
@glennreyes I just use ```type Function = () => void``` to reduce code repeat